tests/unit/models/requests/test_channel_authorize.py

```diff
@@ -56,3 +56,14 @@ def test_has_multiple_signing_methods_is_invalid(self):
                 passphrase=_DUMMY_STRING,
                 seed_hex=_DUMMY_STRING,
             )
+
+    def test_ctor_has_positional_args(self):
```

Comment:
Based on this test name, I'm not entirely sure what the test is trying to accomplish. Can you rename it to something more descriptive? (Not sure if this is the best name because I'm not entirely sure what the intent of the test is)
```suggestion
    def test_throws_if_positional_args_mixed_with_non_positional_args(self):
```

Reply:
this test purports to show that positional arguments cannot be used in the constructor at all.